src/main/resources/scripts/announcements/generate_json.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+import argparse
+import json
+import mysql.connector
```

Comment:
Unfortunately this isn't part of base python either.  You'll have to add instructions for how to install.  Please be sure to use something like a virtualenv.

Reply:
I hope a README would suffice?

Reply:
Yup I think a README would be good.  I think you can definitely assume python & pip, but in general I like when it can be (mostly) foolproof with just giving them the list of commands.

So like:
* run this to create the venv
* run this to activate it
* run this to install the necessary libraries
* Then here's some basic usage for how to run the scripts.

If this was a larger and more regularily used script, I'd say we need a full blown install process (such as the one with the IRIDA uploader https://github.com/phac-nml/irida-uploader/blob/development/Makefile).  In this case since its a "run once and forget it" thing I think just some instructions is fine.

Reply:
Please see cad41a5c5b492472a2c327c888a1e58494b59140.

Reply:
That looks great.  Thanks!